### PR TITLE
README file editing

### PR DESCRIPTION
**Pull request by awiednoor**
- Add the description of the project and all the details to the readme file

### CODE REVIEW

**Review by halitbatur — APPROVED** [review]

**Review by okurezgi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,135 @@
...
+Let's share is an API for people to donate and request availiable
+products and exchanges them among each other, to help pepole from unprivilidged communities.
+
+# ⚙️ Demo
+
+Server link: [Let's Share API]
+
+# 🧱 Features
+
+- Sigin/Signup authentication
```
Comment (khaldarov) [inline README.md:24]: typo here

File: README.md [hunk 2]
```diff
@@ -1 +1,135 @@
...
+- 3rd Party authentication (Google, Facebbok)
```
Comment (khaldarov) [inline README.md:25]: typo here

File: README.md [hunk 3]
```diff
@@ -1 +1,135 @@
...
+- List products
+- CRUD operations for products
+- CRUD operations for users
+- Donate products
+- Request products
+- Filter products by category
+- Search products
+- Middleware for validation and error handling
+
+> The overriding design goal for Markdown's formatting syntax is to make it as readable
```
Comment (khaldarov) [inline README.md]: were these supposed to be deleted?

File: README.md [hunk 4]
```diff
@@ -1 +1,135 @@
...
+
+- 🧔🏻 [Halit Batur]
+- 🧔🏻 [Muslim Omar]
+
+### Developers
+
+- 👩🏻 [Noor Awied]
+- 👩🏻‍🦱 [Ezgi Okur]
+- 👩🏻‍🦱 [Nilay Aydin]
+- 👨🏻 [Adnan Khaldarov]
```
Comment (khaldarov) [inline README.md]: it's only khaldar, but i LOVE the emojis, lol

File: README.md [hunk 5]
```diff
@@ -1 +1,135 @@
...
+- 🧔🏻 [Amjad Khatib]
+
+# 🛠 Tech
+
+Let's Share uses a number of open source projects to work properly:
+
+- [ExpressJS] - fast node.js network app framework [@tjholowaychuk]
```
Comment (khaldarov) [inline README.md]: are you tagging a user here?

File: README.md [hunk 6]
```diff
@@ -1 +1,135 @@
...
+2. Create your Feature Branch (`git checkout -b feature/AmazingFeature`)
+3. Commit your Changes (`git commit -m 'Add some AmazingFeature'`)
+4. Push to the Branch (`git push origin feature/AmazingFeature`)
+5. Open a Pull Request
+
+# 🔑 License
+
+MIT
+
+**Free Software, Hell Yeah!**
```
Comment (khaldarov) [inline README.md:122]: everyone loves free software. it looks great. great work @awiednoor.

**Pull request merged**